lib/jwt_signed_request/verify.rb

```diff
@@ -92,5 +93,25 @@ def verified_headers?
         Headers.fetch(header_key, request) == header_value
       end
     end
+
+    def parsed_claims_path
+      @parsed_claims_path ||= URI.parse(claims['path'])
+    end
+
+    def standard_query_values(path)
+      URI.decode_www_form(path.query).sort if path && path.query
+    end
```

Comment:
So this breaks up the query params in a collection of tuples.
eg:
`[["foo", "1"], ["bar", "2"]]`
and then sorts the key..

makes sense

Reply:
The other option you could do is transform these query params to a hash and check the equality that way as it doesn't depend on ordering.

To convert query params to a hash you can use:
```ruby

Rack::Utils.parse_nested_query("foo=1&bar=2") 
```

Reply:
its prob a better way as it saves you time doing a sort twice

Reply:
problem with this is that query strings can contain the same key multiple times.

Reply:
can always use `CGI#parse` instead to handle the duplicate keys

```ruby
CGI.parse("a=1&a=2")
=> {"a"=>["1", "2"]}
```

Reply:
but then u have the issue with ordering as well.. looks like `URI.decode_www_form(path.query).sort` is the best option